From df0d76045dfae08e89f1e67009239c5341949b33 Mon Sep 17 00:00:00 2001
Message-Id: <df0d76045dfae08e89f1e67009239c5341949b33.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Fri, 31 Oct 2014 12:16:08 +0000
Subject: [PATCH 09/10] MUST_REBASE [VPG]: drm/i915: Fix multiple warnings
 introduced by shmem statics patch

Fix multiple fumbles in:

commit d72db0cab270a69bdf5a59dac131172c6b990022
Author: Sourab Gupta <sourab.gupta@intel.com>
Date:   Mon Aug 11 13:12:04 2014 +0530

    MUST_REBASE [VPG]: drm/i915: Sysfs interface to get GFX shmem usage stats per process

For: GMINL-1778
Change-Id: Ideeb97f22e3aae0e7eb44616413458a5373a352e
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/i915_gem.c       |    1 -
 drivers/gpu/drm/i915/i915_gem_debug.c |    7 ++++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_gem.c b/drivers/gpu/drm/i915/i915_gem.c
index baae06e..8b1ef30 100644
--- a/drivers/gpu/drm/i915/i915_gem.c
+++ b/drivers/gpu/drm/i915/i915_gem.c
@@ -4606,7 +4606,6 @@ i915_gem_close_object(struct drm_gem_object *gem_obj,
 			struct drm_file *file_priv)
 {
 	struct drm_i915_gem_object *obj = to_intel_bo(gem_obj);
-	int ret;
 
 	i915_gem_obj_remove_pid(obj);
 }
diff --git a/drivers/gpu/drm/i915/i915_gem_debug.c b/drivers/gpu/drm/i915/i915_gem_debug.c
index e61a9cc..3f551eb 100644
--- a/drivers/gpu/drm/i915/i915_gem_debug.c
+++ b/drivers/gpu/drm/i915/i915_gem_debug.c
@@ -327,8 +327,8 @@ void i915_gem_obj_remove_pid(struct drm_i915_gem_object *obj)
 	mutex_unlock(&obj->base.dev->struct_mutex);
 
 	if (found == 0)
-		DRM_DEBUG("Couldn't find matching tgid %d for obj 0x%x\n",
-				current_tgid, (u32) obj);
+		DRM_DEBUG("Couldn't find matching tgid %d for obj %p\n",
+				current_tgid, obj);
 }
 
 void i915_gem_obj_remove_all_pids(struct drm_i915_gem_object *obj)
@@ -699,7 +699,8 @@ __i915_get_drm_clients_info(struct drm_i915_error_state_buf *m,
 	struct pid_stat_entry *pid_entry, *temp_entry;
 	struct pid_stat_entry *new_pid_entry, *new_temp_entry;
 	struct list_head per_pid_stats, sorted_pid_stats;
-	int ret = 0, total_shared_prop_space = 0, total_priv_space = 0;
+	int ret = 0;
+	size_t total_shared_prop_space = 0, total_priv_space = 0;
 
 	INIT_LIST_HEAD(&per_pid_stats);
 	INIT_LIST_HEAD(&sorted_pid_stats);
-- 
1.7.9.5

